nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/PutMongoRecordIT.java

```diff
@@ -191,4 +197,33 @@ public void testInsertNestedRecords() throws Exception {
         assertEquals(4, collection.count());
         //assertEquals(doc, collection.find().first());
     }
+
+    @Test
+    public void testArrayConversion() throws Exception {
+        TestRunner runner = init(PutMongoRecord.class);
+        MockSchemaRegistry registry = new MockSchemaRegistry();
+        String rawSchema = "{\"type\":\"record\",\"name\":\"Test\",\"fields\":[{\"name\":\"nom\",\"type\":\"string\"," +
```

Comment:
Let's get rid of the "doc" fields here, I'm sure just pasted from from the example use case that illustrated the bug, but still quite unnecessary and tends to clutter up the test.

Reply:
Also I think let's change the names of fields and values unless we have permission from the original user to use such test data.

Reply:
Yeah, good idea. I'll change them up.